build.gradle

```diff
@@ -1,20 +1,23 @@
 plugins {
     id "org.jetbrains.intellij" version "0.3.7"
+    id "com.github.hierynomus.license" version "0.14.0"
     id "jacoco"
 }
 
 repositories {
     mavenCentral()
 }
 
-apply plugin: 'org.jetbrains.intellij'
-
 intellij {
     version '2017.3'
     plugins 'IntelliLang'
     pluginName 'LSP Support'
 }
 
+license {
+    header rootProject.file('resources/HEADER')
```

Comment:
Add this also pls (can't push to your fork)

```groovy    
    strictCheck = true
    exclude "**/*.properties"
    exclude "**/*.xml"
```

Reply:
Added